vmdb/app/models/miq_request.rb

```diff
@@ -221,19 +221,17 @@ def build_default_approval
   def self.requests_for_userid(userid)
     requester = User.find_by_userid(userid)
     return [] unless requester
-    find_all_by_requester_id(requester.id)
+    where(:requester_id => requester.id)
   end
 
   def self.all_requesters(conditions = nil)
-    find(:all,
-         :conditions => conditions,
-         :select     => "requester_id, requester_name",
-         :group      => "requester_id, requester_name",
-         :include    => "requester"
-    ).inject({}) do |h, r|
-      h[r.requester_id] = (r.requester.nil? ? "#{r.requester_name} (no longer exists)" : r.requester_name)
-      h
-    end
+    where(conditions)
+      .select("requester_id, requester_name")
+      .group("requester_id, requester_name")
+      .includes("requester")
+      .each_with_object({}) do |r, h|
```

Comment:
Yeah, I did a couple of these, either because I spotted them, or because they were upsetting rubocop. But whether I was trying to get the modified lines rubocop clean, or just no-worse-than-before, fluctuated a bit over the adventure.